src/components/Reports/ReportsPartialsCampaigns/ReportsPartialsCampaigns.js

```diff
@@ -189,7 +197,9 @@ const ReportsPartialsCampaigns = ({ location, dependencies: { dopplerApiClient }
                             </li>
                             <li>
                               <FormattedMessage id="reports_partials_campaigns.last_click_date" />{' '}
-                              <span>{state.campaignSummaryResults.lastClickDate}</span>
+                              <span>
+                                <FormattedDate value={state.campaignSummaryResults.lastClickDate} />
```

Comment:
Dates are nice now, and other numbers?

![image](https://user-images.githubusercontent.com/1157864/79365633-3fd82680-7f21-11ea-82ea-435d5936b9fd.png)

I mean, are we not using thousand separators?

Reply:
changed

Reply:
Thanks, but then I realized that we are not formatting numbers in reports 🤦‍♂️, sorry.

Maybe we decided to not format them?

Well, decide it with the team, sorry by the noise.